### DISCUSSION TITLE
Beginner looking to contribute to the project

### DISCUSSION THREAD

**Discussion by leoAshu** [dubinc/dub#489] [General]
Hi everyone, 
I am a Full Stack Developer with 3+ years of experience. I'm just starting out to contribute to open source. I was exploring repositories when I came across this one. I'd really like to dig into the codebase and contribute towards it. 
I am hoping to get some pointers on how to begin, some good first issues or the best practices involved here. Any suggestion would be helpful. Thank you.

**Comment by steven-tey** [comment]
Thanks for the question @leoAshu! We recently released a [self-hosting guide](https://dub.sh/self-host) – feel free to give that a read to learn how to set up Dub.co locally. Then, you can start contributing to the codebase by looking through some of the [open issues](https://dub.sh/issues) and see which one you're interested in solving :)

**Reply by emapeire** [reply]
That's great! I will review the guide to get more involved in this project.

**Reply by leoAshu** [reply]
Hi @steven-tey,
Thanks a lot for the prompt response! I appreciate it. I'll definitely check out the self-hosting guide for Dub.co and get the local setup rolling. Excited to dive into the codebase!
Also, I'll explore the open issues to find something aligned with my skills and interests. 
If there's any specific area you'd recommend for a beginner like me, do let me know. 
Looking forward to contributing and being a part of the Dub.co community!
